.cirrus.yml

```diff
@@ -18,6 +18,7 @@ task:
     - cargo doc --all --no-deps --features=abi-7-21
   test_script:
     - . $HOME/.cargo/env
+    - export SKIP_DEVICE_TESTS=1
```

Comment:
I've fixed this. Now there's just the issue that `test_mount_unmount` hangs in CI on MacOS. This is the problem with writing tests, unfortunately sometimes you find bugs :crying_cat_face:.  It's strange because the integration tests still pass...